dotnet/src/SemanticKernel.Abstractions/SemanticFunctions/IPromptTemplate.cs

```diff
@@ -17,7 +17,7 @@ public interface IPromptTemplate
     /// Get the list of parameters required by the template, using configuration and template info.
     /// </summary>
     /// <returns>List of parameters</returns>
-    IList<ParameterView> GetParameters();
+    IReadOnlyList<ParameterView> GetParameters();
```

Comment:
In another PR we will have `GetFunctionViews`, we should have consistent naming so either use `GetParameterViews` here or `GetFunctions` in the other PR. I would prefer the shorter function names.

Reply:
Actually, in this case, I think we can just replace with a property accessor. This list should not change, ever.

Reply:
Done!